### ISSUE TITLE
Heading structure incorrect

### ISSUE THREAD

**Issue by PimDerks** [smashingmagazine/redesign#285]
Headings like "Latest Posts" and "Popular Posts" are `h4`, while the items below them are or a higher level. Below "Latest posts" the articles have `h2`'s as the title. Below "Popular posts" there are even multiple `h1`'s.

The "Latest/Popular Posts" headings should be `h2` and the articles after them should be `h3`. An incorrect heading structure will result in AT (and thus its users) not being able to correctly understand the structure of the page.

**Comment by vitalyfriedman** [comment]
Good catch, @PimDerks — will resolve the issue as soon as I can!
